### ISSUE TITLE
ResistanceCalculator.java can be removed

### ISSUE THREAD

**Issue by ysthakur** [blair-robot-project/robot2023#9]
You guys can probably remove [`ResistanceCalculator.java`](https://github.com/blair-robot-project/robot2023/blob/main/src/main/java/frc/team449/ResistanceCalculator.java) (and any associated code). It was only added in so that [this PR](https://github.com/wpilibsuite/allwpilib/pull/4162) could be tested (it's closed now). If you guys do have free time and want to test out the resistance calculator, there's a new [PR](https://github.com/wpilibsuite/allwpilib/pull/5500) where you can post your data. I'm sure it would be much appreciated there.

Also, great job with Voyager!

**Comment by calcmogul** [comment]
Indeed, testing that PR on a real control system would be greatly appreciated. That's the main blocker for merge.
